src/util/ChartUtils.ts

```diff
@@ -251,6 +251,13 @@ export const getLegendProps = ({
     item: legendItem,
   };
 };
+
+export type BarSize = {
+  barSize: number | string;
```

Comment:
The name seems misleading. BarSize contains a barSize, but also a stackList? 

Not much better, but maybe 
```suggestion
export type BarSetup = {
  barSize: number | string;
```